api/app/controllers/mno_enterprise/jpi/v1/admin/organizations_controller.rb

```diff
@@ -188,6 +211,15 @@ def user_params
       params.require(:user).permit(:email, :name, :surname, :phone)
     end
 
+    def requested_user_in_org(organization)
+      user = organization.users.find { |u| u.id == params.require(:member)[:id] }
```

Comment:
could you please add some comment on why you try first with id and then by email ?